aether-common-library/aether/common/conf/urls.py

```diff
@@ -61,39 +61,43 @@ def generate_urlpatterns(token=False, kernel=False, app=[]):  # pragma: no cover
 
     else:
         from django.contrib.auth import views
+        from aether.common.auth.forms import get_auth_form
 
-        login_view = views.LoginView.as_view(template_name=settings.LOGIN_TEMPLATE)
+        login_view = views.LoginView.as_view(
+            template_name=settings.LOGIN_TEMPLATE,
+            authentication_form=get_auth_form(),
+        )
         logout_view = views.LogoutView.as_view(template_name=settings.LOGGED_OUT_TEMPLATE)
 
     auth_views = [
-        path('login/', view=login_view, name='login'),
-        path('logout/', view=logout_view, name='logout'),
+        path(route='login/', view=login_view, name='login'),
+        path(route='logout/', view=logout_view, name='logout'),
```

Comment:
Does the logout view hit the Keycloak session logout endpoint? Or is that only be required for OIDC?

Reply:
Yes, but only when there is a keycloak token within the session values. Otherwise there is no way of telling keycloak which user is rtying to log out.

Take a look at the signal:
https://github.com/eHealthAfrica/aether/pull/533/files?file-filters%5B%5D=.py#diff-ff424d29dd76ca35340d7d00829f59f0R126